File: HospitalMascotas/src/app/modelos/clinica.ts
export class Clinica {
    private nombre: String;
    public getNombre(): String {
        return this.nombre;
    }
    public setNombre(value: String) {
        this.nombre = value;
    }
	private direccion: String;
    public getDireccion(): String {
        return this.direccion;
    }
    public setDireccion(value: String) {
        this.direccion = value;
    }
}

File: ttps-spring-clinicamascotas/src/ttps/spring/model/Clinica.java
package ttps.spring.model;

import javax.persistence.*;

import org.springframework.stereotype.Component;
@Entity
@Component
public class Clinica {
	@Id @GeneratedValue
	private long id;
	private String nombre;
	private String direccion;
	public Clinica() {
		
	}
	public String getNombre() {
		return nombre;
	}
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	public String getDireccion() {
		return direccion;
	}
	public void setDireccion(String direccion) {
		this.direccion = direccion;
	}
	public long getId() {
		return id;
	}
	public void setId(long id) {
		this.id = id;
	}
}
